extide/libs.gradle/manifest.mf

```diff
@@ -2,4 +2,4 @@ Manifest-Version: 1.0
 AutoUpdate-Show-In-Client: false
 OpenIDE-Module: org.netbeans.modules.libs.gradle/7
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/libs/gradle/Bundle.properties
-OpenIDE-Module-Specification-Version: 7.5
+OpenIDE-Module-Specification-Version: 7.4.1
```

Comment:
This looks strange - was this bumped by the automatic spec increment? Is the intention behind the ".1" at the end to ensure, that spec version bump on release does not modify the core version (the first two components)?

Reply:
Yes. You are right on that one.

Reply:
This might break when autoupdating installations that already contain org.netbeans.modules.libs.gradle/7 version 7.5: the library will not be upgraded.

Reply:
What's the version available on 12.6 update center? Is the 13 version be higher than the 12.6 one? 

If we have a mess in versioning just in the `master` branch, we can live with it. If we messed up versioning in releases, then I suggest to give up on 1:1 mapping between gradle and NetBeans module versions.

Reply:
Spec version in 12.6 is 7.3, spec version in 13 is 7.4. It's only been 7.5 on master for a short time since I incremented the spec versions at branch. I think @lkishalmi has the right idea on this one.

Reply:
Good - thanks.